filesystems/gcs/src/main/scala/cromwell/filesystems/gcs/GcsPathBuilder.scala

```diff
@@ -247,4 +222,11 @@ case class GcsPath private[gcs](nioPath: NioPath,
     case gcsPath: CloudStoragePath => gcsPath
     case _ => throw new RuntimeException(s"Internal path was not a cloud storage path: $nioPath")
   }
+
+  private def runOnEc[A](io: IO[A])(implicit ec: ExecutionContext): IO[A] = for {
```

Comment:
ToL couldn't you replace this helper with just the call to `IO.shift`?

Reply:
Hmm not sure I see what you mean 🤔